src/parser/serializer.ts

```diff
@@ -16,11 +16,20 @@ import { isAnyArrayBuffer, isDate, isMap, isRegExp, isUint8Array } from './utils
 
 /** @public */
 export interface SerializeOptions {
-  /** the serializer will check if keys are valid. */
+  /**
+   * the serializer will check if keys are valid.
+   * @defaultValue `false`
+   **/
   checkKeys?: boolean;
-  /** serialize the javascript functions **(default:false)**. */
+  /**
+   * serialize the javascript functions
+   * @defaultValue `false`
+   **/
   serializeFunctions?: boolean;
-  /** serialize will not emit undefined fields **(default:true)** */
+  /**
+   * serialize will not emit undefined fields
+   * @defaultValue `true`
+   **/
```

Comment:
https://jira.mongodb.org/browse/NODE-5248

https://github.com/mongodb/node-mongodb-native/blob/main/src/bson.ts#L105-L116

Given that, at this time, we don't have a way of breaking a part the docs, should we document the driver defaults here to?